modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc
+
+[id="virt-adding-kernel-arguments-to-enable-iommu-for-the-bios_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
+.Prerequisites
+* Administrative privilege to a working {product-title} cluster.
+* Intel or AMD CPU hardware.
+* Enable the hardware virtualization extensions for Intel Virtualization Technology (Intel VT-X). AMD-V extensions are typically enabled by default.
++
+[NOTE]
+====
+If you have an AMD CPU, specify the kernel argument as `amd_iommu=on`.
+====
+
+.Procedure
+. Create a `MachineConfig` object file that identifies the kernel argument. The following example shows a kernel argument for an Intel CPU.
+
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker <1>
+    name: 100-worker-iommu <2>
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+    kernelArguments:
+        - intel_iommu=on <3>
+...
+----
+<1> Applies the new kernel argument only to worker nodes.
+<2> The `name` indicates the ranking of this kernel argument (100) among the machine configs and its purpose.
+<3> Identifies the kernel argument as `intel_iommu` for an Intel CPU.
+
+. Create the new `MachineConfig` object file:
++
+[source,terminal]
+----
+$ oc create -f 100-worker-kernel-arg-iommu.yaml
+----
+
+.Verification
+
+. Verify that the new `MachineConfig` object was added.
++
+[source,terminal]
+----
+$ oc get MachineConfig
+----
++
```

Comment:
@lmandavi This is the extra `+` that needs to be deleted, which is throwing off the leveloffset for future modules.

Reply:
Updated the content by removing the extra + which adjusted the TOC presentation.